README.md

```diff
@@ -37,3 +37,5 @@ calculated for all values.
 In the case where there is no indexes behind (index = 0) or only one index behind (index = 1), the program 
 uses special if-statements that either return the array when it has only has that one element [0] or when 
 the array has only the two elements [0, 1].
+The invariant is fibArr[i] = fibArr[i - 2] + fibArr[i - 1].
```

Comment:
That's not an invariant, that's how you compute the result...

Reply:
Apologies, the invariant is that for all numbers greater than or equal to three, each number is the sum of the two previous numbers.

Reply:
That also doesn't really sound like an invariant. What does it tell you about a particular iteration $i$?

Reply:
The $i$ represents the index, so the fib($i$) function calculates the Fibonacci number for that index based on the last two indexes. So, at the beginning of each recursive call to the fib() function, the Fibonacci number is computed and returned.

Reply:
That sounds like your invariant is that the result is already available before the function has run.

Reply:
I meant it more as we know the last two numbers of the Fibonacci sequence, so through each iteration of the loop we are able to find the next Fibonacci number using these two known numbers.

Reply:
Ok, then say that.